x/valset-pref/msg_server_test.go

```diff
@@ -170,38 +223,56 @@ func (suite *KeeperTestSuite) TestDelegateToValidatorSet() {
 			msgServer := valPref.NewMsgServerImpl(suite.App.ValidatorSetPreferenceKeeper)
 			c := sdk.WrapSDKContext(suite.Ctx)
 
-			// if validatorSetExist no need to refund and setValSet again
-			if !test.valSetExists {
-				suite.FundAcc(test.delegator, amountToFund)
+			suite.FundAcc(test.delegator, amountToFund)
 
+			// if validatorSetExist no need to refund and setValSet again
```

Comment:
What do you mean by "refund" here?

Reply:
this comment is obsolete actually. I was using `validatorSetExist` to check if delegator has valset or not, but now i'm using actually setting it based on what the user provides with `setValSet`